packages/web-app-files/tests/unit/components/SideBar/Links/PublicLinks/LinkEdit.spec.js

```diff
@@ -0,0 +1,531 @@
+
+      it('should be required', () => {
+        expect(expirationDatePickerFieldElement.exists()).toBeTruthy()
+        expect(expirationDatePickerFieldElement.attributes('label')).toBe(
+          'Expiration date (required)'
+        )
+      })
+
+      it('should have max-datetime attribute with tomorrow datetime value', () => {
+        expect(expirationDatePickerFieldElement.props().maxDatetime).toBe(
+          DateTime.now()
+            .plus({ days: 2 })
+            .endOf('day')
+            .toISO()
+        )
+      })
+      it('should set expire date when input event is emitted', () => {
+        const wrapper = getMountedWrapper(
+          getLinkCapabilities({
+            enforcedExpireDate: true,
+            days: 2
+          })
+        )
+        const expectedExpireDate = '2021-07-15'
+        const expirationDatePickerFieldElement = wrapper.find(selectors.linkExpireDatePicker)
+        expirationDatePickerFieldElement.element.value = expectedExpireDate
+        expirationDatePickerFieldElement.trigger('input')
+        expect(wrapper.vm.expireDate.target.value).toBe(expectedExpireDate)
+      })
+    })
+    it('should have min-datetime attribute with value one day ahead from provided day', () => {
+      const wrapper = getShallowMountedWrapper()
+
+      const expirationDatePickerElement = wrapper.find(selectors.linkExpireDatePicker)
+      expect(expirationDatePickerElement.props().minDatetime).toBe(
+        DateTime.now()
+          .plus({ days: 1 })
+          .endOf('day')
+          .toISO()
+      )
+    })
+    it('should not be required if public expire date is not enforced', () => {
+      const wrapper = getShallowMountedWrapper()
+
+      const expirationDatePickerFieldElement = wrapper.find(selectors.linkExpireDatePicker)
+
+      expect(expirationDatePickerFieldElement.exists()).toBeTruthy()
+      expect(expirationDatePickerFieldElement.attributes('label')).toBe('Expiration date')
+    })
+    it('should be pre populated if public link in edit has expire date', () => {
+      const wrapper = getShallowMountedWrapper({
+        linkInEdit: {
+          expireDate: '2020-02-20'
+        },
+        ...getLinkCapabilities({ enabledExpireDate: true })
+      })
+      const expirationDatePickerFieldElement = wrapper.find(selectors.linkExpireDatePicker)
+
+      expect(expirationDatePickerFieldElement.props().date).toBe('2020-02-20')
+    })
+  })
+  describe('expiration date delete button', () => {
+    it('should be present if expiration date is not enforced and link in edit has expire date', () => {
+      const wrapper = getShallowMountedWrapper({
+        linkInEdit: {
+          expireDate: '2020-02-20'
+        }
+      })
+      const expirationDateDeleteButtonElement = wrapper.find(selectors.linkExpireDateDeleteButton)
+
+      expect(expirationDateDeleteButtonElement.exists()).toBeTruthy()
+    })
+
+    it('should set expire date to null if clicked', async () => {
```

Comment:
`null` can be confusing as you are asserting `''`
```suggestion
    it('should set expire date to empty string if clicked', async () => {
```